app/views/templates/section_home.html.haml

```diff
@@ -2,3 +2,12 @@
   - breadcrumb :public_node, node
 
   = render partial: 'templates/content_body', object: node.content_body
+
+  - if @section.topics.any?
+    %h4 Popular now
```

Comment:
Maybe this should say 'Information and services' ? I think that's the important bit rather than 'popular now' (popularity isn't implemented yet anyway).